cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/FlushToStorageCommand.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.segmentstore;
+
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.utils.AdminSegmentHelper;
+import io.pravega.shared.protocol.netty.PravegaNodeUri;
+import io.pravega.shared.protocol.netty.WireCommands;
+import lombok.Cleanup;
+import org.apache.curator.framework.CuratorFramework;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Executes a FlushToStorage request against the chosen Segment Store instance.
+ */
+public class FlushToStorageCommand extends ContainerCommand {
+
+    private static final int REQUEST_TIMEOUT_SECONDS = 10;
+
+    /**
+     * Creates new instance of the FlushToStorageCommand.
+     *
+     * @param args The arguments for the command.
+     */
+    public FlushToStorageCommand(CommandArgs args) {
+        super(args);
+    }
+
+    @Override
+    public void execute() throws Exception {
+        ensureArgCount(2);
+
+        final int containerId = getIntArg(0);
+        final String segmentStoreHost = getArg(1);
+        @Cleanup
+        CuratorFramework zkClient = createZKClient();
+        @Cleanup
+        AdminSegmentHelper adminSegmentHelper = (AdminSegmentHelper) instantiateSegmentHelper(zkClient);
+        CompletableFuture<WireCommands.StorageFlush> reply = adminSegmentHelper.flushToStorage(containerId,
+                new PravegaNodeUri(segmentStoreHost, getServiceConfig().getAdminGatewayPort()), super.authHelper.retrieveMasterToken());
+        reply.get(REQUEST_TIMEOUT_SECONDS, TimeUnit.SECONDS);
+        output("Flushed the Segment Container with containerId %d to Storage successfully.", containerId);
```

Comment:
somehow the word `successfully` doesn't make for fluent English in this sentence. Consider removing.

Reply:
Changed it.